src/calculateRentalCost.js

```diff
@@ -4,7 +4,19 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
```

Comment:
The function name 'calculateRentalCost' is descriptive, which is good. However, the parameter 'days' should be more descriptive to indicate its purpose, such as 'numberOfDays'.